lib/Sema/TypeCheckError.cpp

```diff
@@ -1061,7 +1061,9 @@ class Context {
       insertLoc = loc;
       highlight = e->getSourceRange();
       
-      if (InterpolatedString && e->getCalledValue()->getBaseName() ==
+      if (InterpolatedString &&
+          e->getCalledValue() &&
+          e->getCalledValue()->getBaseName() ==
           TC.Context.Id_appendInterpolation) {
```

Comment:
Nitpick: Getting both sides of the comparison in `e->getCalledValue()->getBaseName() == TC.Context.Id_appendInterpolation` on one line would probably read more clearly, and I don't think it would exceed the line limit.

Reply:
Nope, doesn't fit :)